### ISSUE TITLE
<MGT Heroic>

### ISSUE THREAD

**Issue by alexuktc** [dalaranwow/dalaran-wow#5172]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Currently every mob killed in MGT Heroic will respawn in 2 hours since its corpse has disappeared. Including bosses. This allows you to farm the Swift White Hawkstrider. 

When the first 2 bosses respawn so do the doors that block further progress in the instance. This does not happen for the third boss. 
**Expected behaviour**: Tell us what should happen instead
Once killed a mob or a hero in a heroic should not respawn until ID reset (next day).

**Steps to reproduce the problem**:

1.  Kill stuff in MGT Heroic.
2.  Wait 2 hours since corpse disappears.
3.  They will respawn.

**Include proofs for this behaviour**
I do not need proof that it should not happen. Any sort of video to prove this is not a convenient source of proof. Kill something and then go back in 2 hours. 

P.S.
Not sure how to communicate directly with you. I used to be a tester for a TBC server so I know my stuff. I have several stuff about TBC raids (since you mentioned you will rehaul them). Do I list everything in 1 ticket as a compendium or do a separate ticket for each. Do I list all issues in BT as 1 ticket or each boss individually?

**Comment by alexuktc** [comment]
Update:

This is not about MGT, but it is the same issue so I am posting it here. 

The same happens in Sethekk Halls. The trash and bosses respawn. I have not tested it with a timer, but I assume it is 2 hours as well. The Anzu stone was up and I could use it. It played the animation but no Anzu spawned.

**Comment by Aetharon** [comment]
Perhaps this issue is relevant to the ID extension issue that happens with these two instances ( Issues #5045 and #1866 ). I can confirm the extension still happens, at least for Sethekk Halls. As for the respawn of the bosses and trash, I can only assume that is what I've noticed going inside on the days the ID extended - everything, even the Raven's Claw, but without the possibility of summoning Anzu again. It is quite likely that this respawning behaviour is at the base of the extending IDs.

**Comment by TeridanDW** [comment]
Hello,
@Aetharon I don't think the ID extension issue is related to this one.

Regarding:

> The same happens in Sethekk Halls. The trash and bosses respawn. I have not tested it with a timer, but I assume it is 2 hours as well. The Anzu stone was up and I could use it. It played the animation but no Anzu spawned.

Maybe @robinsch can look on this as he worked on Anzu encounter.

Regarding:

> Currently every mob killed in MGT Heroic will respawn in 2 hours since its corpse has disappeared.

I actually think this is correct and intended in a way, I will explain why.
There is a mechanic on retail, for dungeons and raids, where all the trash respawn before boss A is "linked" to him, trash before boss B is linked to him and etc. 
Example - Players enter a random dungeon/raid and clear the trash until the second boss, but some reason doesn't kill it. After few hours they come back to finish it. What they see is no trash and no first boss respawn, but the trash between the first and second boss is now respawned as the group left the second boss alive before.

You can read more about instance mapping in TC bug tracker within issue 20108

Regarding:

> Including bosses. This allows you to farm the Swift White Hawkstrider.

Obviously shouldn't happen.

Regarding:

> Not sure how to communicate directly with you. I used to be a tester for a TBC server so I know my stuff.

If you are truly interested in contributing your knowledge and free time to the project feel free to write an application on dalaran.recruitment@gmail.com as written here http://www.dalaran-wow.com/news/112/reporting-for-duty/ and here http://www.dalaran-wow.com/news/89/recruiting-testers/.

Regarding:

>  I have several stuff about TBC raids (since you mentioned you will rehaul them). Do I list everything in 1 ticket as a compendium or do a separate ticket for each.

I am not sure how @Lothloryen will proceed with TBC raid issues (complete rewrite from 0 or use something else), but we had a massive report (in 1 bug report) for Naxxramas and in my eyes it wan't very good and readable at all. So 1 bug report for each boss encounter would be better and 1 big bug report for the entire trash (spells, events, etc) for each raid.
